Util/swarning.m

```diff
@@ -0,0 +1,82 @@
+function swarning(varargin)
```

Comment:
This should be included in one of the relevant commit!

Reply:
done, made into the first commit of import functions